source/develop/release-management/features/virt/engine-dedicated-cpus.md

```diff
@@ -36,27 +36,22 @@ Engine:
 1. Add the new policies to CPU Pinning Policy enum
 2. Save host capabilities CPU topology in vds_dynamic
 3. Hold hostToAvailableCpu data structure:
-
-    Map Host id -> availableCpus object
-
-    availableCpus:
-
-    Map socket-id -> List of availableCores Object 
-
-    Socket: socket nr
-
-    availableCoresObject:
-    
-    Contains list of available cpus in the core
+    it will be on this form Map<Integer,Map<Integer,Map<Integer,List<String>>>>
+    Host id -> Map of Socket id to Map of Core id to list of cpus
+   (assuming host id, core id, socket id is unique)
 
     **Save/Update cached data structure:**    
 
     Option 1:
     * On updateVDSStatisticsData update the data structure with host capabilities
+      * On stop host, the host should be removed, and all the VM pinnings to that host should be pinned to other hosts
+      * On start host, the host should be added
```

Comment:
what does stop/start mean? stop- maintenance? removed from the cluster? start - host in UP status?

do we really need this information? If a host becomes irrelevant the VMs will migrate away, and if they have a CPU pinning policy they go to your point of migration in this doc.

Reply:
so you are saying this will be handled by the scheduler and this shouldn't matter for us anyway, right?

Reply:
Yes

Reply:
Done, removed those lines as it should be obvious that it is handled by the scheduler